<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomDialog</class>
 <widget class="QDialog" name="RoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>538</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="wdg_tilte" native="true">
       <property name="minimumSize">
        <size>
         <width>561</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>41</height>
        </size>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>561</width>
          <height>23</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lb_title">
           <property name="minimumSize">
            <size>
             <width>201</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>201</width>
             <height>21</height>
            </size>
           </property>
           <property name="text">
            <string>房间号：11111111</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>128</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_min">
           <property name="minimumSize">
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>一</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_max">
           <property name="minimumSize">
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_X">
           <property name="minimumSize">
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QWidget" name="wdg_left" native="true">
         <property name="minimumSize">
          <size>
           <width>431</width>
           <height>261</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>11111111</width>
           <height>11111111</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="UserShowForm" name="wdg_userShow" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wdg_right" native="true">
         <property name="minimumSize">
          <size>
           <width>151</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>151</width>
           <height>300</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>140</width>
               <height>289</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QWidget" name="wdg_list" native="true">
                <property name="minimumSize">
                 <size>
                  <width>122</width>
                  <height>16</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>213</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="wdg_button" native="true">
       <property name="minimumSize">
        <size>
         <width>561</width>
         <height>41</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>41</height>
        </size>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>551</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="cb_video">
           <property name="minimumSize">
            <size>
             <width>73</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>73</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>视频</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_auvio">
           <property name="minimumSize">
            <size>
             <width>73</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>73</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>音频</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_desk">
           <property name="minimumSize">
            <size>
             <width>73</width>
             <height>18</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>73</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string>桌面</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_moo">
           <property name="minimumSize">
            <size>
             <width>72</width>
             <height>22</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>72</width>
             <height>22</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>滤镜</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>帽子</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>猪头</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pb_exit">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>31</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>31</height>
            </size>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserShowForm</class>
   <extends>QWidget</extends>
   <header>usershowform.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
